.travis.yml

```diff
@@ -14,9 +14,10 @@ branches:
   - master
   - development
 script:
-- git config --global user.email "arnab@binary.com"
-- git config --global user.name "Arnab Karmakar"
-- yarn deploy-gh-pages
+  - git config --global user.email "arnab@binary.com"
+  - git config --global user.name "Arnab Karmakar"
+  - yarn deploy-gh-pages
+  - yarn test-browserstack
```

Comment:
@arnabk I had some reasons for adding it after the deploy, but now it makes more sense to add it before deploy.